Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM registry.opensuse.org/opensuse/leap:latest
+
+ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1 UWSGI_WSGI_FILE=/pcw/webui/wsgi.py UWSGI_MASTER=1
+ENV UWSGI_HTTP_AUTO_CHUNKED=1 UWSGI_HTTP_KEEPALIVE=1 UWSGI_LAZY_APPS=1 UWSGI_WSGI_ENV_BEHAVIOR=holy
+
+COPY . /pcw/
+WORKDIR /pcw
```

Comment:
Those lines better to go later as they more likely to change than the zypper cmds. wdyt?

Reply:
I agree here with Yianni. I suggest:
 1. Run `zypper`
 2. Copy `requirements.txt`
 3. Run `pip`
 4. Copy the rest.

Reply:
I updated the Dockerfile accordingly, however it is highly unlikely that any package interferes with the custom `/pcw` directory.